src/js/controllers/preferences.js

```diff
@@ -89,9 +89,6 @@ angular.module('copayApp.controllers').controller('preferencesController',
         value: $scope.wallet.balanceHidden
       };
 
-      if (wallet.isPrivKeyExternal)
-        $scope.externalSource = wallet.getPrivKeyExternalSourceName() == 'ledger' ? 'Ledger' : 'Trezor';
```

Comment:
Yep, security options are available for wallets which have the privkey locally not externally. So these wallets cannot sign, encrypt, request fingerprint, access to 12 words, etc